### PR TITLE
Update to include new binaries for Auth0 SPA JS

### PR DESCRIPTION
**Pull request by frederikprijck**
This PR includes a new version of SPA JS that is not released. It is just a draft version of something that might be improving the behavior.

Would you be possible to see if you are able to reproduce it with this version?

I also removed supertokens as that js file was not included and failing the branch for me, and I also made the expireToken function use the a0Config, to make it easier to use a different client.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -27,8 +27,6 @@ function login() {
     })
 }
 
-const lock = new supertokenslock.getNewInstance()
```
Comment (smogg) [inline app.js:30]: Apologies for the leftover mess 🙇‍♂️

**Review by smogg — COMMENTED** [review]
Thank you for the fix. I am no longer able to reproduce the problem with the demo on this branch. We are pulling the compiled js file and delivering it to our production. I'll reply tomorrow if that fixed the problem our users were seeing.

It would be great to be able to review the fix on https://github.com/auth0/auth0-spa-js when it's read.
Appreciated!

File: app.js (hunk 1 above)
Comment (frederikprijck) [reply app.js:30]: No worries about that!

**Comment by frederikprijck** [comment]
When we hear back from you and it solves your user's problems, we will work on getting a fix in the master branch. I will ping you once that is merged and released.

**Comment by smogg** [comment]
@frederikprijck we tested with a small group and it seems to have helped. Of course those might have been false positives, given that the bug only occurs _sometimes_, so we will now roll out to more users. I will report back in a few days. Thanks again!

**Comment by frederikprijck** [comment]
I can imagine it needs a few days to have a better idea on whether or not it is fixed. We will work on fixing it in master in the meantime, even if turns out not to be a bulletproof fix, it should still be a drastic improvement for anyone hitting the error in a comparable use-case.

**Comment by smogg** [comment]
@frederikprijck unfortunatelly, not much has changed. We are still seeing the error, and we are still able to reproduce it in our playground by affecting the network speed in Chrome devtools. We are currently working with the SPA library directly and we will try to submit issues/PRs there as we learn more.

**Comment by frederikprijck** [comment]
Could you let me know how you can reproduce it by affecting the network speed?
I tried with the network speed as well and wasnt able to reproduce it, so I am interested to know how you reproduce it so we can do some further investigation as well.

**Pull request closed without merging**